NEWS.md

```diff
@@ -1,5 +1,12 @@
 # datawizard 0.10.1
 
+BREAKING CHANGES
+
+* Arguments named `group` or `group_by` will be deprecated in a future release.
```

Comment:
Why don't we deprecate now?

Reply:
For a soft transition in our packages. Else, submitting a package with changes would cause warnings in rev-dep checks.